railties/test/rails_info_controller_test.rb

```diff
@@ -14,16 +14,39 @@ class InfoControllerTest < ActionController::TestCase
 
   def setup
     Rails.application.routes.draw do
+      namespace :test do
+        get :nested_route, to: "test#show"
+      end
       get "/rails/info/properties" => "rails/info#properties"
-      get "/rails/info/routes"     => "rails/info#routes"
+      get "/rails/info/routes" => "rails/info#routes"
+      post "/rails/:test/properties" => "rails/info#properties"
+      put "/rails/:test/named_properties" => "rails/info#properties", as: "named_rails_info_properties"
     end
     @routes = Rails.application.routes
 
+    # ie.
+    #
+    # Name                            Path                                        Verb        Controller#Action
+    # ---------------------------------------------------------------------------------------------------------------
+    # "test_nested_route"             "/test/nested_route(.:format)"              "GET"       "test/test#show"
+    # "rails_info_properties"         "/rails/info/properties(.:format)"          "GET"       "rails/info#properties"
+    # "rails_info_routes"             "/rails/info/routes(.:format)"              "GET"       "rails/info#routes"
+    # ""                              "/rails/:test/properties(.:format)"         "POST"      "rails/info#properties"
+    # "named_rails_info_properties"   "/rails/:test/named_properties(.:format)"   "PUT"       "rails/info#properties"
+
```

Comment:
I'm not sure what is this comment about. Maybe it is misplaced?

Reply:
Hey, thanks for checking in. This comment makes it easier for me to understand all the routes that have been set up in this test as documentation (which are subsequently asserted against below) however it's not necessary.

Reply:
Let's remove then because it will very likely code will change but this comment will not.

Reply:
Removed